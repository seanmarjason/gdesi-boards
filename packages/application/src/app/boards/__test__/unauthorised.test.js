import { render } from '@testing-library/react'
import Unauthorised from '../unauthorised';
 
it('Unauthorised renders correctly', () => {
  const { container } = render(<Unauthorised />)
  expect(container).toMatchSnapshot()
})
