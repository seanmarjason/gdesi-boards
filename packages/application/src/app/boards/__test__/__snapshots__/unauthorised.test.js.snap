// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unauthorised renders correctly 1`] = `
<div>
  <div
    class="MuiBox-root css-k008qs"
  >
    <main
      class="MuiBox-root css-i9gxme"
    >
      <div
        class="MuiStack-root css-13klx5x-MuiStack-root"
      >
        <h6
          class="MuiTypography-root MuiTypography-h6 css-1miy0lu-MuiTypography-root"
          element="h1"
        >
          Not authorised to access this page
        </h6>
      </div>
    </main>
  </div>
</div>
`;
